src/hydro/hydro_system.hpp

```diff
@@ -67,15 +72,21 @@ template <typename problem_t> class HydroSystem : public HyperbolicSystem<proble
 		x2Velocity_index,
 		x3Velocity_index,
 		pressure_index,
-		primEint_index,	  // auxiliary internal energy (rho * e)
-		primScalar0_index // first passive scalar (only present if nscalars > 0!)
+		primEint_index,	   // auxiliary internal energy (rho * e)
+		primScalar0_index, // first passive scalar (only present if nscalars > 0!)
+		// TODO(benwibking): = check what is enabled
+		x2Magnetic_index, // when magnetic fields exist, then we also store cc-ave of the two orthogonal b-fields, so they can be reconstructed to the
```

Comment:
Is this TODO still needed, or can we delete it?

Reply:
I don't remember adding this comment. I think it can be removed. I think @AstroKriel originally wrote this code?

Reply:
OK. @AstroKriel, can you have a look at this comment and see if it can safely be removed?

Reply:
I vaguely remember adding this comment a while ago as a reminder about a something we chatted about in a quokka-telecom: we might want to reorder MHD and passive scalars indices, since when we're only doing MHD, then we're currently carrying around an unused scalar field (primScalar0_index).